1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,234 @@
+---
+RFC:  RFCnnnn
+Author:  travisez13
+Status:  Draft
+SupercededBy:  N/A
+Version: 0.1
+Area: Engine
+Comments Due: 6/30/2019
+---
+
+# `PowerShell Core` Policy
+
+## Motivation
+
+Consumers, developers, and enterprise system administrators should be able to flexibly and reliable configure PowerShell 7.
+
+## Acknowledgement
+
+I based this off of @iSazonov 's RFC, for just a slightly different purpose.
+[PR #111](https://github.com/PowerShell/PowerShell-RFC/pull/111)
+
+## Goals
+
+1. Specify how PowerShell 7 will deal with having both Windows PowerShell and PowerShell Core Group Policy.
+    - This is covered in [Policy settings Setting Fall-Back](#policy-settings-setting-fall-back).
+1. Correct how the `pwsh -settingsfile` switch behaves.
+    - This is covered in  [Parameter `-settingsfile`](#Parameter--settingsfile)
+
+## Definitions
+
+- **Computer-Wide settings/policy**  - setting or policy applied to the entire operating system.
+- **User settings/policy** - setting or policy applied only to the user.
```

Comment:
applied to a specific user of an OSE, not applied to the OSE as a whole

Reply:
it would be nice if you used the suggestions feature.
```suggestion
- **User settings/policy** - setting or policy applied to a specific user of an OSE, and not applied to the OSE as a whole.
```

I think this language is very technical.